src/main/java/edu/harvard/iq/dataverse/globus/GlobusServiceBean.java

```diff
@@ -209,6 +233,24 @@ public JsonObject requestAccessiblePaths(String principal, Dataset dataset, int
         }
         //The dir for the dataset's data exists, so try to request permission for the principal
         int requestPermStatus = requestPermission(endpoint, dataset, permissions);
+        
+        if (requestPermStatus == 409) {
+            // This is a special case - a 409 *may* mean that the rule already
+            // exists for this endnote and for this user (if, for example, 
+            // Dataverse failed to remove it after the last upload has completed). 
+            // That should be ok with us (but let's confirm that is indeed the 
+            // case; alternatively it may mean that permissions cannot be issued 
+            // for some other reason):
+            String ruleId = getRuleId(endpoint, principal, "rw");
+            if (ruleId != null) {
+                requestPermStatus = 201;
```

Comment:
Should this get logged? Something went wrong with a prior removal (which doesn't have to stop this request but shouldn't be ignored as it's a bug/problem if perms aren't going away)